module/documents/active-effect.mjs

```diff
@@ -615,6 +615,24 @@ export default class ActiveEffect5e extends ActiveEffect {
 
   /* -------------------------------------------- */
 
+  /**
+   * Add help icon & tooltip to Attribute Key header
+   * @param {jQuery|HTMLElement} html  The Active Effect Config HTML.
+   */
+  static onRenderActiveEffectConfig(html) {
+    html = html instanceof HTMLElement ? html : html[0];
+    const helpIconElement = document.createElement("i");
+    helpIconElement.classList.add("fa-solid", "fa-circle-question");
+    const tooltipText = game.i18n.localize("DND5E.ActiveEffectAttributeKeyTooltip");
+    helpIconElement.setAttribute("data-tooltip", tooltipText);
+    helpIconElement.setAttribute("data-tooltip-direction", "RIGHT");
+    helpIconElement.setAttribute("data-locked", true);
+    const targetElement = html.querySelector("section[data-tab='effects'],section[data-tab='changes'] .key");
```

Comment:
This sticks it in the wrong spot in V12, you can add `.key` back to the first part or use `:is` to grab either:

```suggestion
    const targetElement = html.querySelector("section:is([data-tab='effects'],[data-tab='changes']) .key");
```

or

```suggestion
    const targetElement = html.querySelector("section[data-tab='effects'] .key,section[data-tab='changes'] .key");
```